### PR TITLE
moved the info-text to inside of the results container

### PR DESCRIPTION
**Pull request by mhigbyflowers**
very small change

### CODE REVIEW

File: public-ui/src/app/views/survey-view/survey-view.component.html [hunk 1]
```diff
@@ -185,7 +185,8 @@ <h2 *ngIf="!loading" class="secondary-display">
               </div>
             </div>
           </div>
-  </div>
+        </div>
```
Comment (SrushtiGangireddy) [inline public-ui/src/app/views/survey-view/survey-view.component.html:189]: This should actually be at the top. I am not sure how it got to the bottom. 
This should be below the survey name or this is not relevant anymore.

Comment (mhigbyflowers) [reply public-ui/src/app/views/survey-view/survey-view.component.html:189]: image

Comment (mhigbyflowers) [reply public-ui/src/app/views/survey-view/survey-view.component.html:189]: thanks

**Review by SrushtiGangireddy — APPROVED** [review]
Looks good. thanks.

**Pull request merged**